packages/dts-gen/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@chevrotain/dts-gen",
+  "version": "8.0.1",
+  "description": "Generates type definitions for chevrotain CST nodes",
+  "keywords": [],
+  "bugs": {
+    "url": "https://github.com/Chevrotain/chevrotain/issues"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Shahar Soel"
+  },
+  "typings": "lib/src/api.d.ts",
+  "main": "lib/src/api.js",
+  "files": [
+    "lib/src/**/*"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Chevrotain/chevrotain.git"
+  },
+  "scripts": {
+    "---------- CI FLOWS --------": "",
+    "build": "npm-run-all clean compile",
+    "test": "npm-run-all coverage",
+    "---------- BUILD STEPS --------": "",
+    "clean": "shx rm -rf dev",
+    "compile:watch": "tsc -w",
+    "compile": "tsc",
+    "coverage": "nyc mocha"
```

Comment:
we would probably want a script here to update the snapshots
maybe even call it on a pre-commmit hook in the root package.json

Reply:
yeah, the thought occurred to me after creating the PR.  probably not a pre-commit hook since we'd need the compiled library